src/index.vue

```diff
@@ -15,28 +18,59 @@ export default {
     },
     preventDefault: {
       type: Boolean
-    }
+    },
+    config: {
+      type: Array,
+      default: null
+    }    
   },
   mounted() {
-    window.addEventListener(this.event, this.emitEvent);
+    this.m_keyListeners = []    
+    if (this.config) {
+      // use multi-key config exclusively
+      this.config.forEach(config => {
+        config.events.forEach(event => {
+          let listener = this.emitEvent(config.keyCodes, config.modifiers, config.preventDefault)
+          this.m_keyListeners.push({event, listener})
+          window.addEventListener(event, listener);  
+        });    
+      })    
+    } else {
+      let listener = this.emitEvent(this.keyCode, this.modifiers, this.preventDefault)
+      this.m_keyListeners.push({event: this.event, listener})
+      window.addEventListener(this.event, listener);  
+    }
   },
   destroyed() {
-    window.removeEventListener(this.event, this.emitEvent);
+    this.m_keyListeners.forEach(({event, listener}) => {
+      window.removeEventListener(event, listener);
+    })
   },
   methods: {
-    emitEvent(e) {
-      if (event.keyCode === this.keyCode || !this.keyCode) {
-        if (this.preventDefault){
-            e.preventDefault();
-        }
-        // Check if all modifiers were clicked and return, if not
-        if (this.modifiers.length) {
-          for (const modifier of this.modifiers) {
-            if (!event[modifier]) return
+    emitEvent(keyCodes, modifiers, preventDefaults) {
+      return (e) => {
+        const index = keyCodes.indexOf(event.keyCode)
```

Comment:
This breaks in single-key mode because `keyCodes` is not an array.

Reply:
Thanks, should be fixed now.